src/components/app/index.tsx

```diff
@@ -26,10 +32,17 @@ type Image = {
   type: ImageType,
   options: CodecOptions,
   data?: ImageBitmap,
+  dataUrl: string,
```

Comment:
Rename to `downloadUrl`? `dataUrl` suggests `data:…`.

Reply:
yeah - I can get behind that.